### ISSUE TITLE
Limit show_prediction only to few classes

### ISSUE THREAD

**Issue by casertap** [TeamHG-Memex/eli5#149]
I have 300 target classes and showing explanation of all of them make my browser very slow.
it could be very good if show_prediction could show the explanation of the 3 best predictions and not all the classes (y).
How can I do that?

**Comment by lopuhin** [comment]
@casertap you can provide a list of targets to show via the "targets" argument: http://eli5.readthedocs.io/en/latest/autodocs/eli5.html#eli5.explain_prediction - so right now you'll have to first manually select top 3 best predictions and then pass them to explain_prediction via "targets" (make sure it has the same format as "target_names", if you are using them).

But having the ability to just pass "top_targets=3" looks like a cool feature!

**Comment by kmike** [comment]
+1, I wanted 'top_targets=1' more than once

**Comment by lopuhin** [comment]
Shall we rename ``top`` to ``top_features`` for consistency?

**Comment by kmike** [comment]
> Shall we rename top to top_features for consistency?

Hm, a good question. For explain_weights `top` name sounds fine - "top weights"; for explain_prediction less so. I'm +0 to keep is as `top`.
